docs/ref/forms/api.txt

```diff
@@ -822,6 +822,9 @@ classes, as needed. The HTML will look something like:
     >>> f["subject"].legend_tag(attrs={"class": "foo"})
     <legend for="id_subject" class="foo required">Subject:</legend>
 
+You may further modify the rendering of form rows by using a
+:ref:`custom BoundField <custom-boundfield>`.
```

Comment:
I don't know how to add the double backticks to `BoundField` here. I always get spell checking errors if I try doing it.

Reply:
Not sure you can 🤔 I'd...

1. Add BoundField to spelling.txt
2. Ask @felixxm if there's a better way 😅

Reply:
Thanks! I think it should be OK.

Here's `docs/faq/admin.txt`:

```
:meth:`~django.contrib.admin.ModelAdmin.save_model` hook, you can update an
object to reflect the user that edited it. See :ref:`the documentation on
ModelAdmin methods <model-admin-methods>` for an example.
```

Reply:
> This doesn't seem to be necessary...

Great! If it's not triggering an error, don't add it 👍